migrations/00013_shows_seasons_tags.sql

```diff
@@ -0,0 +1,244 @@
+
+COMMENT ON SEQUENCE "public"."shows_tags_id_seq"  IS NULL;
+
+--- END CREATE SEQUENCE "public"."shows_tags_id_seq" ---
+
+--- BEGIN CREATE SEQUENCE "public"."seasons_tags_id_seq" ---
+
+
+CREATE SEQUENCE IF NOT EXISTS "public"."seasons_tags_id_seq"
+	INCREMENT BY 1
+	MINVALUE 1
+	MAXVALUE 2147483647
+	START WITH 1
+	CACHE 1
+	NO CYCLE;
+
+ALTER SEQUENCE "public"."seasons_tags_id_seq" OWNER TO builder;
+GRANT SELECT ON SEQUENCE "public"."seasons_tags_id_seq" TO builder; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
+GRANT USAGE ON SEQUENCE "public"."seasons_tags_id_seq" TO builder; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
+GRANT UPDATE ON SEQUENCE "public"."seasons_tags_id_seq" TO builder; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
+
+COMMENT ON SEQUENCE "public"."seasons_tags_id_seq"  IS NULL;
+
+--- END CREATE SEQUENCE "public"."seasons_tags_id_seq" ---
+
+--- BEGIN CREATE TABLE "public"."shows_tags" ---
+
+CREATE TABLE IF NOT EXISTS "public"."shows_tags" (
+	"id" int4 NOT NULL DEFAULT nextval('shows_tags_id_seq'::regclass) ,
+	"shows_id" int4 NULL  ,
+	"tags_id" int4 NULL  ,
+	CONSTRAINT "shows_tags_pkey" PRIMARY KEY (id) ,
+	CONSTRAINT "shows_tags_tags_id_foreign" FOREIGN KEY (tags_id) REFERENCES tags(id) ON DELETE CASCADE ,
+	CONSTRAINT "shows_tags_shows_id_foreign" FOREIGN KEY (shows_id) REFERENCES shows(id) ON DELETE CASCADE
+);
+
+ALTER TABLE IF EXISTS "public"."shows_tags" OWNER TO builder;
+
+GRANT SELECT ON TABLE "public"."shows_tags" TO builder; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
+GRANT INSERT ON TABLE "public"."shows_tags" TO builder; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
+GRANT UPDATE ON TABLE "public"."shows_tags" TO builder; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
+GRANT DELETE ON TABLE "public"."shows_tags" TO builder; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
+GRANT TRUNCATE ON TABLE "public"."shows_tags" TO builder; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
+GRANT REFERENCES ON TABLE "public"."shows_tags" TO builder; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
+GRANT TRIGGER ON TABLE "public"."shows_tags" TO builder; --WARN: Grant\Revoke privileges to a role can occure in a sql error during execution if role is missing to the target database!
```

Comment:
This are likely not needed because in GCP builder does the migrations and will thus be the owner (fyi)